lib/index.js

```diff
@@ -226,6 +226,26 @@ const KiteAPI = {
       .catch(_ => []);
   },
 
+  validateFileForRelatedCode(filename) {
+    return this.request(
+      {
+        path: '/codenav/validate',
+        method: 'POST',
+      },
+      JSON.stringify({ filename })
+    );
+  },
+
+  requestRelatedCode(filename, line, selection, editor) {
+    return this.request(
+      {
+        path: '/clientapi/editor/related',
```

Comment:
@tonycheang should be `/codenav/editor/related` per the spec

Reply:
The spec changed from review. New PR for this: https://github.com/kiteco/kite-api-js/pull/46